common-app-lib/src/cmr/common_app/services/search.clj

```diff
@@ -41,14 +42,18 @@
   "Returns a cache backed by Redis. This cache is used to store a map of cmr scroll-ids to ES scroll-ids 
    in a consistent way acrosss all instances of search."
   []
-  (redis-cache/create-redis-cache {:ttl (/ (scroll-id-cache-ttl) 1000)}))
+  (redis-cache/create-redis-cache {:ttl (/ (scroll-id-cache-ttl) 1000)
```

Comment:
I know you did not do this, but should 1000 be a def?

Reply:
Even better, since no other cashes use these time to live values I was able to just remove dividing by 1000 and change the original value from milliseconds to seconds.